### PR TITLE
feat: integrate LIT decrypt

### LINKED ISSUE #222: Decrypt emails
**User Story:**
As a RM engineer
I want to decrypt emails received in applications
So that round operators can see each project's email address

**Acceptance Criteria**
- RM displays decrypted application email on application detail page

**Product & Design Links:**

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- adds try catch error handling to LIT
- introduces answerBlocks to state
- add ability to run through application answers and decrypt answers if the user has round operator role

##### Refers/Fixes

closes #222 

##### Testing

URL to test decrypt : [HERE](http://localhost:3000/#/round/0xb91fec0b68f39cbfdd75e4f08042c60724e1bd3b/application/0x3600000000000000000000000000000000000000000000000000000000000000-0xb91fec0b68f39cbfdd75e4f08042c60724e1bd3b)

Deploy

note: reached out to LIT team to ask if

-  we can pass in a config to override the alert box and show our own custom message
https://discord.com/channels/896185694857343026/916383445784096839/1011148394938519613

- how to get jest to work with lit (this issue is something even grants hub is facing)
https://discord.com/channels/896185694857343026/916383445784096839/1011162956349845515

### CODE REVIEW

File: packages/round-manager/src/features/round/ViewApplicationPage.tsx [hunk 1]
```diff
@@ -46,6 +53,67 @@ export default function ViewApplicationPage() {
     }),
   })
 
+
+  const [answerBlocks, setAnswerBlocks] = useState<AnswerBlock[]>();
```
Comment (thelostone-mc) [inline packages/round-manager/src/features/round/ViewApplicationPage.tsx:57]: @chibie as you had suggested I used useEffect cause but I'm unclear on how to ensure this gets called once after the application.answers have loaded. `answerBlocks ` gets overwritten multiple times and that prevents me from showing the needed data 

The decryption piece works without any issues

cc @shavinac / @dschinkel would love if you guys had some ideas here

URL: http://localhost:3000/#/round/0xb91fec0b68f39cbfdd75e4f08042c60724e1bd3b/application/0x3600000000000000000000000000000000000000000000000000000000000000-0xb91fec0b68f39cbfdd75e4f08042c60724e1bd3b

The IPFS ptr: https://gitcoin.mypinata.cloud/ipfs/bafkreif6f7e2jaa7445jflaintyav5m63gutxl6umpgvxsjhdqsqv326za

**Review by thelostone-mc — COMMENTED** [review]
stuck in using effects on react.

File: packages/round-manager/src/features/round/ViewApplicationPage.tsx (hunk 1 above)
Comment (thelostone-mc) [reply packages/round-manager/src/features/round/ViewApplicationPage.tsx:57]: Thanks @shavinac for helping me out here.
I've cherry-picked your commit onto my branch

File: packages/round-manager/src/features/round/ViewApplicationPage.tsx [hunk 2]
```diff
@@ -250,13 +311,15 @@ export default function ViewApplicationPage() {
...
 
               <h2 className="text-xs mb-2">Funding Profit</h2>
-              <p className="text-base mb-6">{ getAnswer("Profit2022", application) }</p>
+              <p className="text-base mb-6">{ getAnswer("Profit2022") }</p>
 
               <h2 className="text-xs mb-2">Team Size</h2>
-              <p className="text-base mb-6">{ getAnswer("Team Size", application) }</p>
+              <p className="text-base mb-6">{ getAnswer("Team Size") }</p>
+
+              <p className="text-base mb-6">{getAnswer("Is your project cool enough to apply ?")}</p>
```
Comment (shavinac) [inline packages/round-manager/src/features/round/ViewApplicationPage.tsx:322]: is this a real application question? o.0

Comment (chibie) [reply packages/round-manager/src/features/round/ViewApplicationPage.tsx:322]: nah it's not! it should be removed

File: packages/round-manager/src/features/round/ViewApplicationPage.tsx [hunk 3]
```diff
@@ -221,11 +282,11 @@ export default function ViewApplicationPage() {
               <div className="grid sm:grid-cols-3 gap-2 md:gap-10">
                 <div className="text-grey-500 truncate block">
                   <MailIcon className="inline-flex h-4 w-4 text-grey-500 mr-1"/>
-                  <span className="text-xs text-grey-400">{ getAnswer("Email", application) }</span>
+                  <span className="text-xs text-grey-400">{ getAnswer("Email") }</span>
```
Comment (chibie) [inline packages/round-manager/src/features/round/ViewApplicationPage.tsx:285]: can all the question be defined in `ApplicationQuestions` enum so here could be `ApplicationQuestions.EMAIL`?

**Review by chibie — CHANGES_REQUESTED** [review]

File: packages/round-manager/src/features/round/ViewApplicationPage.tsx (hunk 2 above)
Comment (thelostone-mc) [reply packages/round-manager/src/features/round/ViewApplicationPage.tsx:322]:
```suggestion
```

**Review by chibie — APPROVED** [review]
lgtm!

**Pull request merged**